src/main/java/uk/gov/hmcts/probate/controller/DocumentController.java

```diff
@@ -49,6 +49,8 @@
 
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 import static uk.gov.hmcts.probate.model.Constants.LONDON;
+import static uk.gov.hmcts.probate.model.Constants.YES;
```

Comment:
please compare this java file with master , dont see any of the changes made here are required . These are ununsed imports and should fail in checkstyles itself when u build